### ISSUE TITLE
Accessibility / Useability: Click target should be the same size as a button

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#15]
1. Go to:http://data.ottawa.ca/user/login
2. Clickable area on menu buttons should be the same size as the button.

Resolution: Increase clickable area to match the size of the button. See below graphic showing smaller accessible area for clicking.

![click target](https://cloud.githubusercontent.com/assets/5271253/8060797/3c345348-0e95-11e5-8cee-04a6318a741f.JPG)

**Comment by deniszgonjanin** [comment]
@Dawnzar ready for review

**Comment by Dawnzar** [comment]
Retested: resolved.
